Content.Server/DeviceLinking/Systems/SignalTimerSystem.cs

```diff
@@ -85,10 +90,7 @@ private void UpdateTimer()
                 continue;
 
             Trigger(uid, timer);
-
-            if (timer.DoneSound == null)
-                continue;
-            _audio.PlayPvs(timer.DoneSound, uid);
+            break;
```

Comment:
Remove this break, this will prevent any other signal timer from running in the same tick which is unintended behaviour.

Reply:
fix